controllers/nstemplateset/namespaces_test.go

```diff
@@ -533,7 +533,7 @@ func TestEnsureNamespacesFail(t *testing.T) {
 
 }
 
-func TestDeleteNamespsace(t *testing.T) {
+func TestDeleteNamespace(t *testing.T) {
```

Comment:
is there any test-case where you would verify a situation when one namespace is in terminating state - has termination timestamp?

Reply:
Not in here, but [this](https://github.com/codeready-toolchain/toolchain-e2e/pull/336/files#diff-e413827761efc65be1d2b683f71859e11dc041eddacaef444a8fcb11c409aa37R323) checks for NS in terminating state and having deletion timestamp.

Reply:
Also, this particular line is just a typo fix

Reply:
I meant if there is a test-case where you would 
```
// given
create Namespace with a deletion timestamp and add it to the fake client

// when
allDeleted, err := manager.ensureDeleted(logger, nsTmplSet)

// then
require.NoError(t, err)
assert.False(t, allDeleted)
in other words: verify that it's still waiting for the namespace being completely deleted
```

Reply:
added it now @MatousJobanek

Reply:
awesome, thanks :+1: